tailormap-components/projects/core/src/lib/workflow/workflows/EditgeometryWorkflow.ts

```diff
@@ -46,11 +46,11 @@ export class EditgeometryWorkflow extends Workflow {
   }
 
   private openForm(geom: GeoJSONGeometry) {
-
-    const objecttype = this.event.feature.objecttype;
+    const feature = this.event.feature;
+    const objecttype = feature.objecttype;
     const feat = this.featureInitializerService.create(objecttype,
-      {...this.event.feature, geometrie: geom  });
-
+      {...feature, geometrie: geom  });
+    feat.objectGuid = feature.objectGuid;
```

Comment:
If you expect this to happen more in the future, maybe its nice to add as argument to `featureInitializerService.create` (`public create(type: string, params: any, objectGuid = FeatureInitializerService.STUB_OBJECT_GUID_NEW_OBJECT): Feature {`)